### PR TITLE
OptiGAN Modules

### PR DESCRIPTION
**Pull request by guneet19**
Modules for integrating UC Davis OptiGAN model into Gate 10 [WIP]

### CODE REVIEW

File: opengate/tests/src/test074_optigan_root_output_no_conversion.py
Comment (guneet19) [inline opengate/tests/src/test074_optigan_root_output_no_conversion.py:1]: You can ignore this file

File: opengate/tests/src/test074_optigan_phsp_and_kill_actor.py
Comment (guneet19) [inline opengate/tests/src/test074_optigan_phsp_and_kill_actor.py:1]: You can ignore this file.

File: opengate/tests/src/test075_testing_use_optigan_feature.py
Comment (guneet19) [inline opengate/tests/src/test075_testing_use_optigan_feature.py:1]: Users can activate optigan by just using sim.physics_manager.use_optigan = True

**Comment by nkrah** [comment]
This was merged via another branch.

**Pull request closed without merging**